deployments/deploy-ssh.sh

```diff
@@ -314,12 +322,14 @@ spec:
     effect: "NoSchedule"
   affinity:
     nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-        - matchExpressions:
-          - key: ${K8S_VENDOR_KEY}
-            operator: ${K8S_VENDOR_OPERATOR}
-            ${K8S_VENDOR_VALUE}
+      preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Any specific reason to change it from hard requirement to soft one here?

Reply:
yeah, for example there can be test environments where entire k8s cluster is dedicated to run torpedo. Also I had a problem of labeling k8s node with px/enabled false in aws due to intricating authentication process. Anyway preferred node affinity will always put torpedo pod to non px node when such node exists.

Reply:
as per definition if we use `preferredDuringSchedulingIgnoredDuringExecution` it tries to but in case it's not possible it will use even a node which has px, which is something we don't want, we need to stick to guarantee there is a dedicated torpedo node before we deploy torpedo

Reply:
Well, I have a case when I run torpedo tests on signel cluster. By default it gets labeled with px/enabled: true. So I need torpedo to run in any case.